dns/domains.py

```diff
@@ -27,6 +27,7 @@ class Domain(DomainOptional):
     "basalt": { "cname": "pyroxenium.github.io" },
     "brag": { "cname": "bragosmagos.github.io" },
     "cash": { "cname": "mcjack123.github.io" },
+    "ccsimplegui": { "cname": "masongulu.github.io/CCSimpleGUI/" },
```

Comment:
```suggestion
    "ccsimplegui": { "cname": "masongulu.github.io" },
```

This needs to be a domain, not a URL.